components/account/AccountHeader.vue

```diff
@@ -100,8 +100,8 @@ const isSelf = $(useSelfAccount(() => account))
         <div absolute top-18 inset-ie-0 flex gap-2 items-center>
           <AccountMoreButton :account="account" :command="command" />
 
-          <button v-if="!isSelf && relationship?.following" flex gap-1 items-center w-full rounded op75 hover="op100 text-pink" group @click="toggleNotify()">
-            <div rounded-full p2 group-hover="bg-pink/10">
+          <button v-if="!isSelf && relationship?.following" flex gap-1 items-center w-full rounded op75 hover="op100" group :aria-label="`${relationship?.notifying ? 'Stop notifying' : 'Notify'} me when @${account.username} posts`" @click="toggleNotify()">
+            <div rounded-full p2 border-1 :class="relationship?.notifying ? 'text-primary border-primary' : 'border-base'" :group-hover="relationship?.notifying ? 'text-red bg-red/10 border-red' : 'text-primary'">
```

Comment:
can you replace all div inside button with span? it is not valid html, add  `block` attribute

Reply:
great catch! the changes from this PR are going to be added to #1127, so we'll make that change there 😄